applications/CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ add_subdirectory(sam)
 # scripts
 add_subdirectory(script)
 
+if(NOT TUTTLE_PYTHON_VERSION)
```

Comment:
Indeed, but impossible to have 2 conditions in a single CMake if statement... See commit c24d4ea